src/torchcodec/decoders/_video_decoder.py

```diff
@@ -66,6 +67,10 @@ class VideoDecoder:
             probably is. Default: "exact".
             Read more about this parameter in:
             :ref:`sphx_glr_generated_examples_decoding_approximate_mode.py`
+        transforms (sequence of transform objects, optional): Sequence of transforms to be
+            applied to the decoded frames by the decoder itself, in order. Accepts both
+            ``torchcodec.transforms.DecoderTransform`` and ``torchvision.transforms.v2.Transform``
```

Comment:
For it to render as links in the docs:

```suggestion
            :class:`~torchcodec.transforms.DecoderTransform` and :class:`~torchvision.transforms.v2.Transform`
```

We should also create a doc page for the transforms!